fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/ExpressionTranslator.java

```diff
@@ -380,4 +409,12 @@ private static org.apache.doris.analysis.AssertNumRowsElement.Assertion translat
                 throw new AnalysisException("UnSupported type: " + assertion);
         }
     }
+
+    public static GroupByClause.GroupingType translateGroupingType(
+            PhysicalRepeat repeat) {
+        if (repeat instanceof PhysicalGroupingSets) {
+            return GroupingType.GROUPING_SETS;
+        }
+        throw new AnalysisException("UnSupported type");
+    }
```

Comment:
PhysicalRepeat is a Plan, don't translate in ExpressionTranslator.
You can move to PhysicalPlanTranslator.